client/packages/core/src/store.ts

```diff
@@ -66,38 +195,41 @@ function deleteInMap(m, path) {
   deleteInMap(m.get(head), tail);
 }
 
-function setInMap(m, path, value) {
```

Comment:
Updates to use a loop instead of recursion--the profiling is pretty inconsistent, but in my chrome profile it went from ~350ms for 100k entities to ~220ms.

I also experimented with another function that is optimized for setting the full path of a triple so that there was no looping. It seemed faster, but the improvement was within the margin of error for the profile.